src/main/kotlin/no/nav/syfo/sykmelding/user/api/SykmeldingApiV2.kt

```diff
@@ -17,7 +17,7 @@ import no.nav.syfo.sykmelding.status.StatusEventDTO
 fun Route.registrerSykmeldingApiV2(sykmeldingerService: SykmeldingerService) {
     route("api/v2/sykmeldinger") {
         accept(ContentType.Application.Json) {
-            get {
+            get("/") {
```

Comment:
Hvilke konsekvenser har dette for eksisterende konsumenter? Ktor har blitt ganske kresen på trailing slash :/